pages/ref/core-cells.mdx

```diff
@@ -441,6 +441,27 @@ s.skipBits(5);                   // all but first 5 bits
 let fizz: Slice = s.loadBits(1); // load only 1 bit
 ```
 
+## Slice.loadBool
+
+```tact
+extends mutates fun loadBool(self: Slice): Bool;
+```
+
+Extension mutation function for the [`Slice{:tact}`][p].
+
+Loads a single bit and returns a [`Bool{:tact}`][bool] value from the [`Slice{:tact}`][p]. Produces `true{:tact}` if the signed [`Int{:tact}`][int] value of the loaded bit is $-1$, `false{:tact}` otherwise.
```

Comment:
```suggestion
Loads a single bit and returns a [`Bool{:tact}`][bool] value from the [`Slice{:tact}`][p]. Produces `true{:tact}` if the signed [`Int{:tact}`][int] value of the loaded bit is $1$, `false{:tact}` otherwise.
```

Reply:
for some reason I cannot commit this change... I'm wondering if it's because your PR settings

Reply:
Hmm, weird. I didn't change the default ones upon creation of the fork :thinking: 

Anyways, we previously wrote -1 in `.storeBool()` description as it's what community was expecting to see coming from FunC:

![image](https://github.com/tact-lang/tact-docs/assets/68142933/ec207995-3b3e-4989-bc82-ae92db6acdfd)

Should we change -1 to 1 there too?

Reply:
> Should we change -1 to 1 there too?

yes, please: negative one-bit numbers look confusing